### PR TITLE
Jcd/profile page touch up

### PR DESCRIPTION
**Pull request by Jean-Christoffer**
Fetches your posts
fetches posts from users you follow
User can comment, react to posts

### CODE REVIEW

File: html/profile/profile.css [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  
+  .profile-container .header-image-container .profile-image {
+    width: 133px;
+    height: 133px;
+    position: absolute;
+    bottom: -50px;
+    left: 20px;
+    border: 3px solid #212529;
+    z-index: 10;
+    border-radius: 999px;
```
Comment (AnaCascante) [inline html/profile/profile.css]: change to 50 % instead of px.

File: html/profile/profile.html [hunk 2]
```diff
@@ -42,26 +43,25 @@
 </header>
```
Comment (AnaCascante) [inline html/profile/profile.html:43]: add m-3 in line 20 to give some margin (it makes the nav look less compress) :)

**Comment by AnaCascante** [comment]
I like the changes you did :), well done! 
Something I will like to consider is the blue color.... I am not 100%  convinced that this is the best color to use.. we can play with other colors to see the result later ;)

**Comment by Jean-Christoffer** [comment]
Thanks:) will make the requested chsnges then squash and merge

**Pull request merged**